README.md

```diff
@@ -18,7 +18,7 @@ A set of [React](https://reactjs.org/) components created by, and for, [Mavenlin
 - Setup [CSS modules](https://github.com/css-modules/css-modules). One way of doing that is with [`style-loader` for Webpack](https://github.com/webpack-contrib/style-loader) and [`css-loader` for Webpack](https://github.com/webpack-contrib/css-loader).
 - Setup [SVG processing](https://svgontheweb.com/#spriting). One way of doing that is with [`svg-sprite-loader` for Webpack](https://github.com/kisenka/svg-sprite-loader).
 
-  ```js
+  ```js static
```

Comment:
What is the `static` keyword doing?

Reply:
It's part of the React Styleguidist docs: all `js` is rendered into the page (e.g. code examples) but adding `js static` skips the rendering. It is useful for showing code without an actual rendering -- this has confused engineers when reading a guide on how to document a new component. I figured I should keep using `static` to spread the knowledge.